ajax/libs/ice/package.json

```diff
@@ -8,6 +8,13 @@
         "*.js",
         "*.map"
       ]
+    },
+    {
+      "basePath": "lib/es5",
+      "files": [
+        "*.js",
+        "*.map"
```

Comment:
Let's modify from A to B as the followings.
Because if there are some folder be created future, this rule still can work.
- A
```js
  "npmFileMap": [
    {
      "basePath": "lib",
      "files": [
        "*.js",
        "*.map"
      ]
    },
    {
      "basePath": "lib/es5",
      "files": [
        "*.js",
        "*.map"
      ]
    }
  ],
```

- B
```js
  "npmFileMap": [
    {
      "basePath": "lib",
      "files": [
        "**/+(Ice|Glacier)*.+(js|map)"
      ]
    }
  ],
```

Thank you.